src/core/non_wx_functions.h

```diff
@@ -220,6 +298,6 @@ inline float ConvertXYToPhiInDegrees(float x, float y) {
         return rad_2_deg(atan2f(y, x));
 }
 
-inline float ReturnWavelenthInAngstroms(float kV) {
-    return 12.262643247 / sqrtf(kV * 1000 + 0.97845e-6 * powf(kV * 1000, 2));
```

Comment:
I can see that we're making clear that this is float multiplication; are we using 12.2639 instead of the previous number because the level of accuracy isn't greatly affected by the smaller precision?